src/ptwt/_stationary_transform.py

```diff
@@ -14,6 +15,22 @@
 )
 
 
+def _wrap_padding(x: torch.Tensor, paddings: Sequence[int]) -> torch.Tensor:
+    """Pads a tensor in circular mode more than once if needed"""
+    s = x.shape[-1]  # TODO rename s to something more meaningful
+
+    # TODO write what the regular case is
```

Comment:
@NiclasPi would you please address the three TODOs I added in the code for improving internal documentation?